### PR TITLE
Change 'SUSE Enterprise Storage Integration'

### PR DESCRIPTION
**Pull request by svenseeberg**
* Make link to SES documentation independent of version
* Use command tag to mark full user input
* Base64-encode Ceph secret, BSC#1070920
* Fix YAML indentation, BSC#1070920
* Improve titles and text
* Add steps with Ceph Secret to PersistentVolume procedure

### CODE REVIEW

File: xml/admin_integration_ses.xml [hunk 1]
```diff
@@ -268,13 +289,13 @@ spec:
...
 
-<screen>&prompt.user;<command>kubectl delete pod busybox-rbd</command></screen>
+<screen>&prompt.user;<command>kubectl delete pod <replaceable>CONTAINER_NAME</replaceable></command></screen>
 
   <para>
    And when the command finishes, run
   </para>
 
-<screen>&prompt.user;<command>kubectl delete persistentvolume yourpv</command></screen>
+<screen>&prompt.user;<command>kubectl delete persistentvolume <replaceable>PV_NAME</replaceable></command></screen>
```
Comment (sebastian-philipp) [inline xml/admin_integration_ses.xml]: Regarding *Note: Deleting a Pod*, the RBD itself is not deleted, right?

**Comment by sebastian-philipp** [comment]
Would it make sense to either update [this FAQ entry](https://wiki.microfocus.com/index.php/SUSE_CaaS_Platform/FAQ#How_do_I_use_SUSE_Enterprise_Storage_with_a_Storage_Class_for_persistent_storage.3F) or link to the official documentation?

File: xml/admin_integration_ses.xml [hunk 2]
```diff
@@ -30,95 +30,98 @@
...
    </step>
    <step>
     <para>
      Create an image in the SES cluster. On the master node, run the following
      command:
     </para>
-<screen>&prompt.root;<command>rbd create -s <replaceable>1G</replaceable> <replaceable>YOUR_VOLUME</replaceable></command></screen>
+<screen>&prompt.root;<command>rbd create -s <replaceable>SIZE</replaceable> <replaceable>YOUR_VOLUME</replaceable></command></screen>
     <para>
-     Where <replaceable>1G</replaceable> is the size of the image and
```
[the remark this answers is not in the document]
Comment (svenseeberg) [reply xml/admin_integration_ses.xml]: Yes, all replaceables should have an explanation below the example. I'll add that. Thanks for pointing that out.

**Comment by evrardjp** [comment]
Yes, we have many places where documentation overlaps. Updating the FAQ entry to the product documentation would make sense, until the FAQ is ripped apart to only point links to the product documentation (dreaming)...

File: xml/admin_integration_ses.xml (hunk 1 above)
Comment (svenseeberg) [reply xml/admin_integration_ses.xml]: No, AFAICT it would stay. As with the creation, this is managed by Ceph. Adding a sentence.

**Comment by svenseeberg** [comment]
Yes, having everything in the official product documentation is the goal we should aim for. But there always have to be some documents as a source for the documentation. Writing documentation from source code is a little tedious ;-)

**Comment by sebastian-philipp** [comment]
+1 for having a FAQ as long as we don't have multiple contradicting documents on the same topic.

**Comment by r0ckarong** [comment]
I would suggest to replace the FAQ entry with a link to the finished section for this once it's merged. The overall goal is to completely integrate the FAQs (internal and external) into the official docs.

**Comment by r0ckarong** [comment]
@svenseeberg I have added some minor fixes. Otherwise this looks good to me. If there is no further stuff you wish to add please let me know so I can merge.

**Pull request merged**